docs/config/index.md

```diff
@@ -71,7 +71,7 @@ It is important to note that in Vite's API the `command` value is `serve` during
 
 ## Async Config
 
-If the config needs to call async function, it can export a async function instead:
+To use async functions in your configuration, make sure to export the config as an async function:
```

Comment:
I think the original wording is ok, but we could clarify that the async function can be passed to `defineConfig` too. As the snippet doesn't correspond with directly exporting the async function: 
```suggestion
If the config needs to call async functions, it can export an async function instead. And this async function can also be passed through `defineConfig` for improved intellisense support:
```